layouts/partials/mathjax_support.html

```diff
@@ -0,0 +1,22 @@
+<script type="text/javascript" async
+  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
```

Comment:
can you:
- host this file locally (download it and commit it like the other partials including external libs)
- add an enabler in the markdown parameters to prevent this from being loaded everywhere? (you can check the other partials as well)

Reply:
I think it's OK